docs/src/atomistic/engines/lammps.rst

```diff
@@ -0,0 +1,134 @@
+.. _engine-lammps:
+
+LAMMPS
+======
+
+.. list-table::
+   :header-rows: 1
+
+   * - Official website
+     - How is metatensor supported?
+   * - https://lammps.org
+     - In a separate fork
+
+
+Supported model outputs
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Only the :ref:`energy <energy-output>` output is supported in LAMMPS, as a
+custom ``pair_style``. This allows running molecular dynamics simulations with
+interatomic potentials in metatensor format; distributing the simulation over
+multiple nodes and potentially multiple GPUs.
+
+How to install the code
```

Comment:
Instead of "code" maybe "interface"
```suggestion
How to install the interface
```

Reply:
I think code is clearer for end users here